integration/sawtooth_integration/docker/intkey-smoke.yaml

```diff
@@ -49,7 +49,7 @@ services:
     entrypoint: "bash -c \"\
         ./bin/sawtooth admin keygen && \
         ./bin/sawtooth admin genesis && \
-        ./bin/validator -vv\""
```

Comment:
This test does everything that intkey smoke does and more, so unless something very strange happens, intkey smoke won't fail unless this one also does (and the same mostly goes for xo smoke too). Personally I find the test output unpleasantly cluttered as it is, so I would prefer not to duplicate it.